src/renuka/exam1/AddNumbers.java

```diff
@@ -0,0 +1,37 @@
+package renuka.exam1;
+
+public class AddNumbers {
+	
+	int [] add(int[] nums, int target) {
+		int[] output = new int[2];
+        for(int index=0; index <nums.length; index++) {
+        	for(int innerIndex=0; innerIndex <nums.length; innerIndex++) {
```

Comment:
Inner for loop should from i+1, think on this, if you want to connect on Skype then feel free to message.

Take reference of this code
Kangan - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/987/files
Rupali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/991/files
Dipali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/1003/files